### ISSUE TITLE
Error Spotify Premium Required 

### ISSUE THREAD

**Issue by Bhavin-hyperspace** [lufinkey/react-native-spotify#201]
Hello @lufinkey!

Suddenly i am getting this error when i tried to login with premium account credentials.

**Comment by 0duaht** [comment]
+1

**Comment by 0duaht** [comment]
https://developer.spotify.com/community/news/2022/07/15/mobile-streaming-sdks-update/

**Comment by Bhavin-hyperspace** [comment]
Yes i also found that streaming sdk not working from September 1, 2022. So i think to we need to compulsory move to "react-native-spotify-remote".

**Comment by Bhavin-hyperspace** [comment]
@0duaht 
If you have any other solution then let me know.

**Comment by lufinkey** [comment]
Dang that is really unfortunate. I'll go ahead and put a notice on the repo

**Comment by s-maxx** [comment]
It is really a shame, the spotify remote sdk is so restrictive compared to this one, this is very disapointing of them ...
I would be very interested if you have any other solution to stream spotify like we could with this repo ! (not like the remote sdk where you need the spotify app to be always playing etc.)

**Comment by lufinkey** [comment]
Librespot was the main one I knew of, but I don't think it supported
mobile. It may also be broken from this update too
